source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,78 @@
+
+You can set the default permissions granted to System Admins, Team Admins, Channel Admins, Playbook Admins, Guests (if enabled), and All Members. The permissions granted in the System Scheme apply system-wide, meaning:
+
+* **Guests:** If Guest Accounts are enabled, permissions apply to guest users in all channels, in all teams.
+* **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams.
+* **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+* **Playbook Admins:** Permissions apply to all Playbook Admins in all channels, in all teams.
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+Playbooks roles
+---------------
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who belong to a playbook or are a Playbook Admin.
+
+**Participant**
+
+In the context of Playbooks, participants belong to or take part in a run. They can be a member or a Playbook Admin.
+
+**Playbook Admin**
+
+In the context of Playbooks, Playbook Admins are also members, and may have elevated permissions to change playbook and run visibility as well as functional settings. The Playbook Admin role is a default role, and members need to be promoted to the role from within Playbooks. The Playbook Admin role is applied per playbook. They do not have access to the System Console and their privileges are managed by the System Admin.
+
+Manage playbook visibility
+--------------------------
+
+The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, only Playbook Admins can edit playbooks. Team members can view Public and Private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
+4. Select **Save**.
+
+Manage how playbooks are accessed
+---------------------------------
+
+By default, playbooks are Public. This means that any member of the team can view the playbook and associated runs. Playbooks can be converted from Public to Private, but once a playbook is Private it can't be converted back to Public.
```

Comment:
```suggestion
Playbooks are Public. This means that any member of the team can view the playbook and associated runs. Playbooks can be converted from Public to Private, but once a playbook is Private it can't be converted back to Public.
```